include/shell.h

```diff
@@ -37,6 +37,7 @@
 #define CMD_MAXLINE 4096
 #define CMD_MAXCMDS 20
 #define CMD_OLDSIZE 4096
+#define CMD_MAX_HISTORY_FILE_LINES 500
```

Comment:
Use `constexpr` instead and UpperCamelCase for constants is the preferred standard for new stuff.

Reply:
Fixed